data/ecosystems/b/bitcoin.toml

```diff
@@ -645,19 +672,347 @@ url = "https://github.com/bitpay/bitcore-node"
 [[repo]]
 url = "https://github.com/bitsofproof/supernode"
 
+[[repo]]
+url = "https://github.com/BittyTax/BittyTax"
+
+[[repo]]
+url = "https://github.com/block-core/blockcore"
+tags = ["Protocol"]
+
+[[repo]]
+url = "https://github.com/blockchain/libwally-swift"
+
 [[repo]]
 url = "https://github.com/blockchain/My-Wallet"
 tags = ["Wallet"]
 
 [[repo]]
 url = "https://github.com/blockchain/unused-My-Wallet"
 
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-bech32"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-bip39"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-bytewords"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-crypto-base"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-envelope-python"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-envelope-python"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-envelope-rust"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-libs-java"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-libwally-java"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-libwally-swift"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-lifehash"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-lifehash-python"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-shamir"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-sskr"
+
+[[repo]]
+url = "https://github.com/BlockchainCommons/bc-ur"
```

Comment:
There is a Blockchain Commons ecosystem already.  Do you think it's appropriate to make that a direct sub ecosystem of Bitcoin?

Reply:
Ah, I missed that. Yes, I'll amend to make it a sub system. While blockchain commons does have a lot of generalized security work that could be applied to management of any cryptographic keys, all of their protocol specific projects are only Bitcoin related.